site/content/docs/5.3/migration.md

```diff
@@ -33,6 +33,12 @@ If you need more details about the changes, please refer to the [v5.3.2 release]
 - <span class="badge bg-success">New</span> In order to improve accessibility for people suffering from any form of color blindness, we added a visual cue to form elements on error when focused so information does not rely exclusively on color anymore. Although is has no direct impact, you might want to apply this same modification within your custom form controls if you have any.
 - <span class="badge bg-success">New</span> Labels related to mandatory form fields have been improved for better accessibility. A `<span class="visually-hidden"> (required)</span>` element has been added to ensure correct restitution by assistive technologies like screen readers. Please reflect these modifications into your websites.
 
+- <span class="badge bg-success">New</span> Star rating has been slightly changed for accessibility purpose; a visually hidden `span` matching the title attribute was added and color contrast of inactive stars has been updated. Please reflect this modification into your websites. Our documentation also includes a disabled and a readonly state for this form element.
```

Comment:
Missing `readonly` version in the migration guide. Missing deprecated mixin. Please check all changes that can have an impact on users.

Reply:
"Missing readonly version in the migration guide."
=> `Our documentation also includes a disabled **and a readonly state** for this form element.`

It's not enough maybe? Should I include a small section with more details?

Reply:
My bad, missed the sentence about the readonly state. But the deprecated mixin was not here.